src/DSharp.Script/DSharp.Scripts.csproj

```diff
@@ -1,25 +1,67 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$(SolutionDir)NuGetCommon.props" Condition="Exists('$(SolutionDir)NuGetCommon.props')" />
-  
+
   <PropertyGroup>
     <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
-    <ProjectGuid>{8780581F-7C26-4B64-9236-BA1C458DF36E}</ProjectGuid>
     <DefaultItemExcludes>${DefaultItemExcludes};node_modules\**;dist\**;package-lock.json</DefaultItemExcludes>
     <DisableFastUpToDateCheck>true</DisableFastUpToDateCheck>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <JavaScriptTestRoot>.\test</JavaScriptTestRoot>
+    <JavaScriptTestFramework>Jest</JavaScriptTestFramework>
+    <GenerateProgramFile>false</GenerateProgramFile>
   </PropertyGroup>
-  
+
+  <ItemGroup>
+    <!--<ProjectCapability Include="DotNetCoreWeb" />-->
+    <ProjectCapability Include="JavaScript" />
+    <ProjectCapability Include="JavaScriptProject" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <SourceScript Include="src\**\*.js" />
+    <Content Include="src\**\*.js">
+      <TestFramework>Jest</TestFramework>
+    </Content>
+  </ItemGroup>
+
   <ItemGroup>
-    <PackageReference Include="Microsoft.TypeScript.MSBuild" Version="2.9.1" />
+    <None Remove="test\array.spec.js" />
+    <None Remove="test\clearKeys.spec.js" />
+    <None Remove="test\enumerate.spec.js" />
+    <None Remove="test\extend.spec.js" />
+    <None Remove="test\isValue.spec.js" />
+    <None Remove="test\keyCount.spec.js" />
+    <None Remove="test\keyExists.spec.js" />
+    <None Remove="test\keys.spec.js" />
+    <None Remove="test\value.spec.js" />
   </ItemGroup>
 
-  <Target Name="RunScriptBuild" AfterTargets="Build">
-    <RemoveDir Directories="dist" />
-    <Message Importance="high" Text="Restoring dependencies using 'npm'. This may take several minutes..." />
+  <ItemGroup>
+    <Content Include="test\array.spec.js" />
```

Comment:
Worth doing this as a recursive property:
`<Content Include="test\*.spec.js" />`

Reply:
visual studio likes to expand wildcards when used on the standard itemgroups

Reply:
Not in the new SDK :)